pages/docs/quickstart.mdx

```diff
@@ -79,7 +79,7 @@ To do this, you need a GitHub account. If you do not have one yet, you can creat
 | :---- | :----- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | name  | string | The display name for the button                                                                                                                                                      |
 | url   | string | Full url to the resource                                                                                                                                                             |
-| icon  | string | The icon you would like displayed. We support the following icon sets: **Fa** and **Si** from React Icons. You can search available [icons here](https://linkfree.eddiehub.io/icons) |
+| icon  | string | The icon you would like displayed. We support the following icon sets: **Fa** and **Si** from React Icons. You can search available <a href="https://linkfree.eddiehub.io/icons" target="_blank">icons here</a>  |
```

Comment:
Target blank is stripped by GitHub, suggest using markdown

```suggestion
| icon  | string | The icon you would like displayed. We support the following icon sets: **Fa** and **Si** from React Icons. You can search available [icons](https://linkfree.eddiehub.io/icons)  |
```

Reply:
Yes they suggest not to use `blank` also. We are staying within the same app, a new tab is not required